# default parameters for the X8R RECEIVER

PWM_BACKWARDS_MIN: 1480
PWM_BACKWARDS_MAX: 980
PWM_NEUTRAL_MIN: 1498
PWM_NEUTRAL_MAX: 1499
PWM_FORWARD_MIN: 1520
PWM_FORWARD_MAX: 2000
